src/paths/ForwardingNumber.ts

```diff
@@ -0,0 +1,64 @@
+// This is Generated Source.
+import PathSegment from "../PathSegment";
+import ForwardingNumberInfo from "../definitions/ForwardingNumberInfo";
+import PagingResult from "../PagingResult";
+
+export default class ForwardingNumber extends PathSegment {
+    constructor(prv: PathSegment, id?: string, service?) {
+        super("forwarding-number", id, prv, service);
+    }
+
+    /**
+     *  Add New Forwarding Number
+     */
+    post(body: PostBody): Promise<ForwardingNumberInfo> {
+        return this.getService().send({
+          body: body,
+          method: "post",
+          query: undefined,
+          url: this.getEndpoint(true),
+        }).then(function (res) {
+            return res.json();
+        });
```

Comment:
I think it's obviously more convenient to return the final json object for most users.  In order not to affect the convenience of the most users, but still providing the access to the original response for some rare cases, I think maybe we can add an option to control the return type, by default json is returned.

```typescript
post(body: PostBody, opts?:{returnRawResponse: boolean})
```

What do you think?